client/src/backend/containers/records/export-targets/Form.js

```diff
@@ -0,0 +1,96 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { exportTargetsAPI, requests } from "api";
+import { camelCase } from "lodash";
+import Form from "global/components/form";
+import FormContainer from "global/containers/form";
+
+export default function ExportTargetsContainerForm(props) {
+  const { model, onSuccess, options } = props;
+  return (
+    <FormContainer.Form
+      model={model}
+      name={requests.beExportTargetUpdate}
```

Comment:
When I'm reusing a form like this, I'll try to pass in prop that tells the form if it's new or edit and use the appropriate name based on that prop. Alternately, you can check if this.props.model has an ID or not as a way to see if you're creating or updating.